spark/v3.5/spark/src/test/java/org/apache/iceberg/spark/actions/TestRewritePositionDeleteFilesAction.java

```diff
@@ -1075,49 +1075,49 @@ private void checkResult(
       List<DeleteFile> rewrittenDeletes,
       List<DeleteFile> newDeletes,
       int expectedGroups) {
-    assertThat(rewrittenDeletes.size())
+    assertThat(result.rewrittenDeleteFilesCount())
```

Comment:
the `result` contains the results after rewriting position deletes and is the `actual` part of the comparison. `rewrittenDeletes` contains the `expected` information. The same applies for all the other changes below